### PR TITLE
move to paper folder

### PR DESCRIPTION
**Pull request by lheagy**

### CODE REVIEW

File: paper/myst.yml [hunk 1]
```diff
@@ -15,6 +16,7 @@ project:
       email: doug@eos.ubc.ca
       orcid: 0000-0002-4327-2124
   doi: 10.1093/gji/ggad122
+  arxiv: 2211.14677
```
Comment (lheagy) [inline paper/myst.yml:19]: @stevejpurves: is this the right place to put the arxiv link? I don't see it showing up in the preview

File: paper/casingEMpaper.tex [hunk 2]
```diff
@@ -31,16 +31,14 @@
 \maketitle
 
 \begin{summary}
-
```
Comment (lheagy) [inline paper/casingEMpaper.tex:34]: @stevejpurves: one more question for you... it doesn't look like the summary is automatically being pulled. Is there something I need to do here? Thanks for your help!

Comment (lheagy) [reply paper/casingEMpaper.tex:34]: ah, okay, I was able to fix this by moving the summary into a separate "Abstract" file and putting it under a "Summary" heading

Comment (stevejpurves) [reply paper/casingEMpaper.tex:34]: glad you got it working! I'm not an expert on the tex->myst conversion, in myst we tags blocks as "abstract" or add it to frontmatter but I don't know what the equivalent is in tex.

**Pull request merged**